### ISSUE TITLE
Attachments in Discord not showing as links in IRC

### ISSUE THREAD

**Issue by LordAlderaan** [reactiflux/discord-irc#141]
For me pasting images in discord did not result in a *cdn.discordapp.com* link to the image in IRC. 

I found the issue was with the section in [bot.js](https://github.com/reactiflux/discord-irc/blob/master/lib/bot.js) where it tries to identify if there are attachment but tries a .lenght on the attachments objectmap which returns a *undefined* which is considered a false. 

I fixed it in my local copy of bot.js by doing a size instead. I did not test this issue any further as it seems to be working for me now.

Current code in [bot.js](https://github.com/reactiflux/discord-irc/blob/master/lib/bot.js):
```javascript
 if (message.attachments && message.attachments.length) {
```

My fix:
```javascript
if (message.attachments && message.attachments.size) {
```

**Comment by tjb0607** [comment]
this is affecting me as well on discord-irc v2.1.0 on node.js v6.9.0

**Comment by ekmartin** [comment]
discord.js changed `attachments` to a [Collection](https://github.com/hydrabolt/discord.js/blob/f292e7002f6a6ab8e2233f57b53e8f120b951140/src/util/Collection.js) instead of an array, which I missed. Thank you for the report! Should be fixed in version [2.1.1](https://github.com/reactiflux/discord-irc/blob/master/CHANGELOG.md#211---2016-10-21).
